styles/message.mcss

```diff
@@ -81,12 +85,12 @@ Message {
       a.avatar {
         img {
           width: 50px
+          height: 50px
```

Comment:
I'm just gonna leave this how you did it for now. Let's wait for it to break first. Pretty sure all other clients are assuming a square avatar right now anyway.